src/components.jl

```diff
@@ -256,8 +256,28 @@ mutable struct Font
   color::Colorant
 end
 
-"Create a Font from a list of unordered features"
-function font(args...)
+"""
```

Comment:
I like this suggestion for font and text, but I think it is separate from the documentation - would you mind splitting this out?

Reply:
Yes, this was a mistake. Should be fixed in the new commit.